.circleci/config.yml

```diff
@@ -18,7 +18,7 @@ jobs:
           command: mkcert -install && mkcert localhost
           working_directory: ~/.simulacrum/certs
       - node/install:
-          node-version: 18.12.0
+          node-version: 18.17.1
```

Comment:
Is this required?

Reply:
I bumped to the latest because CircleCI was [complaining about newer npm](https://app.circleci.com/pipelines/github/thefrontside/simulacrum/1279/workflows/44aa26eb-ee4d-44f5-bfb1-77e19b9e0fe0/jobs/1369) and would fail the run. Happy to switch to a more appropriate fix if you have suggestions.